src/lib/Tab/URLManagement/URLWildcardsTab.php

```diff
@@ -87,8 +97,20 @@ public function getOrder(): int
      */
     public function renderView(array $parameters): string
     {
-        $urlWildcards = $this->urlWildcardService->loadAll();
+        $currentPage = $this->requestStack->getCurrentRequest()->query->getInt(
+            self::PAGINATION_PARAM_NAME, 1
+        );
+        $limit = $this->configResolver->getParameter('pagination.url_wildcards');
+
+        $pagerfanta = new Pagerfanta(
+            new ArrayAdapter(
+                $this->urlWildcardService->loadAll()
```

Comment:
Did you know that `URLWildcardService::loadAll()` actually has two parameters that in theory should allow proper pagination? :sweat_smile:

Reply:
and yes and no;), because soon there will be PR that will add a search engine to it, so the loadAll function with these parameters will not be used

as for the count function, you're right, I'll add it in a moment :)